vue/components/ui/input-platforme/input-platforme.vue

```diff
@@ -32,19 +62,40 @@ input[type="text"] {
 }
 
 input[type="text"]:hover {
-    border-color: #dddddd;
+    background-color: #ebeef1;
+    border: solid 1px #dfe1e5;
 }
 
 input[type="text"]:focus {
     background-color: $white;
-    border-color: #dddddd;
+    border: solid 2px $aqcua-blue;
     box-shadow: 0px 1px 8px 0px rgba(32, 33, 36, 0.14);
 }
+
+input[type="text"]:disabled {
+    background-color: $pale-grey;
+    border-color: transparent;
+}
+
+.help-text-validation {
+    display: block;
+}
 </style>
 
 <script>
 export const InputPlatforme = {
     name: "input-platforme",
+    computed: {
+        showSuccess() {
+            return this.success && !this.disabled;

```

Comment:
I can understand the reasoning behind this, but let's allow showing messages even when disabled and let the users of this component to decide whether that behavior makes sense or not.